README.md

```diff
@@ -46,7 +46,7 @@ To compile and generate the VSIX for installation, run:
 npm run compile
 ```
 
-2. To run and open the live instance of the extension, open the VsCode Debugger.
+2. To run and open the live instance of the extension, open the VsCode Debugger and make sure you have extension.ts open in the VsCode editor.
```

Comment:
what is the reason for this? isn't there a way to achieve the same results without this requirement?